algorithm/algorithm.md

```diff
@@ -0,0 +1,147 @@
+# MPM Algorithm
+
+## Main Algorithm
+
+The Material Point Method (MPM) algorithm comprises of 4 major parts.
+
+> **Note** 
+> * MPM stores information at the material points and not at the nodes as it is in Finite Element Method. This includes the application of traction at the particles, not at the nodes.
+> * The code is implementing Modified Update Stress Last (MUSF). The other option is to do Update Stress First (USF) or to do traditional Update Stress last. It is done by moving this block of code to the bottom after the material points velocities and displacements have been updated.
```

Comment:
`MUSL` not `MUSF`. Remove the other option part.

Reply:
`This code uses Modified Update Stress Last (MUSL) algorithm` and add reference to MUSL.

Reply:
Where is the reference we want to make?